packages/xdl/src/logs/PackagerLogsStream.ts

```diff
@@ -416,8 +416,10 @@ export default class PackagerLogsStream {
     const moduleName = match[1];
     const relativePath = path.relative(this._projectRoot, originModulePath);
 
+    // todo: check if we need to re-add this header
+    // https://github.com/expo/expo/commit/f1624cace73fc569402e4304b4b494afa053e046#diff-3bc168b28fe19df4560d1324b96cfa84L103
     const DOCS_PAGE_URL =
```

Comment:
I think we might need to change this URL. It's pointing [at a missing header in the FAQ](https://github.com/expo/expo/commit/f1624cace73fc569402e4304b4b494afa053e046#diff-3bc168b28fe19df4560d1324b96cfa84L103).